docs/modules/getting_started/pages/flow-deep-dive.adoc

```diff
@@ -0,0 +1,42 @@
+:imagesdir: ../assets/images
+[[deep-dive-dash]]
+= Flow Deep-Dive dashboard
+
+The {page-component-title} plugin includes a pre-designed flow deep-dive dashboard that displays flow data.  
+The dashboard can help with quality of service (QoS) monitoring and includes panels that display data in the following categories:
+
+* flow interface statistics (throughput and data use by application)
+* SNMP interface statistics (interface throughput and errors/discards)
+* conversations (throughput and data use by conversation)
+* hosts (throughput and data use by host)
+* DCSPs (throughput and data use by Differentiated Services Code Points)
+
+== QoS options
+
+The dashboard exposes nodes and interfaces with flows, that you can filter on in the *Node* and *Interface* drop-down lists.  
+
+You can also filter flows traffic by ToS/DSCP and ECN (explicit congestion notification) for nodes and interfaces with these values. 
+
+Select a DSCP value (or combination of values) from the list for the QoS you want to visualize (class selector, IP precedence, assured forwarding, expedited forwarding). 
+
+The ECN drop-down lets you further filter on this value to see which flows use ECN-capable transports, and congestion reports for them.  
+ECN values appear in the *Data Usage by Application/Conversation/Host/DSCP* panels.
+This works for aggregated and unaggregated flow data. 
+
+[[deep-dive-add]]
+== Add a Flow Deep-Dive dashboard
+
+. Navigate to the home page of your Grafana instance.
+. In the left menu, click the Configuration icon, and click *Plugins*.
+. Click the *Dashboards* tab. 
+. In the list, click *Flow Deep-Dive|OpenNMS Helm*.
++
+The dashboard appears, ready for you to configure. 
+It also appears in the list of dashboards on the home page.
+
+[[deep-dive-configure]]
+== Configure a Flow Deep-Dive dashboard
+
+By default, the Flow Deep-Dive dashboard uses the OpenNMS Flow and OpenNMS Performance data sources. You can further configure the dashboard by creating queries or editing existing panels. 
```

Comment:
Maybe the prerequisite that the "OpenNMS Performance" and  the "OpenNMS Flow" data sources have to be setup can be stated more clearly. Is there documentation on that?

Reply:
IMO, creating datasources for Performance, Flow, Entity, and Postgres are the first things I do when setting up Helm instances. Which now makes me wonder how hard it would be to create a script that would use the Grafana REST API to mass create an instance of each source.  Did I just give myself a side project to investigate?

Reply:
I'll add text that states that data sources have to be set up and a cross-reference to the getting started section that describes how to do that.